### PR TITLE
(Emre) Database Week1 homework added

### PR DESCRIPTION
**Pull request by arasmr**

### CODE REVIEW

File: Week1/homework/world-queries.txt [hunk 1]
```diff
@@ -37,4 +39,199 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
...
 | Alkmaar           |
 +-------------------+
-28 rows in set (0.00 sec)
\ No newline at end of file
+28 rows in set (0.00 sec)
+
+2. What is the population of Rotterdam ?
+
+SELECT Name, Population FROM city 
+WHERE city.Name = 'Rotterdam';
```
Comment (jasongwartz) [inline Week1/homework/world-queries.txt]: 👍

File: Week1/homework/world-queries.txt [hunk 2]
```diff
@@ -37,4 +39,199 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
...
+| Name      | Population |
++-----------+------------+
+| Rotterdam |     593321 |
++-----------+------------+
+1 row in set (0.01 sec)
+
+3. What's the name of all the countries on the continent ‘North America’ ? 
+
+SELECT Name,Continent FROM country 
+WHERE country.Continent = 'North America';
```
Comment (jasongwartz) [inline Week1/homework/world-queries.txt]: 👍

Keep in mind that the question asked you for the name of the countries. It didn't ask you to return the name and continent - just the name.
You got the query exactly correct, but `SELECT Name FROM ...` would be closer to what the question actually asked.

File: Week1/homework/world-queries.txt [hunk 3]
```diff
@@ -37,4 +39,199 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
...
+| Saint Vincent and the Grenadines | North America |
+| Virgin Islands, British          | North America |
+| Virgin Islands, U.S.             | North America |
++----------------------------------+---------------+
+37 rows in set (0.00 sec)
+
+4. What's the top 10 countries by SurfaceArea ?
+
+SELECT Name,SurfaceArea FROM country 
+ORDER BY SurfaceArea DESC LIMIT 10;
```
Comment (jasongwartz) [inline Week1/homework/world-queries.txt]: 👍

File: Week1/homework/world-queries.txt [hunk 4]
```diff
@@ -37,4 +39,199 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
...
+| India              |  3287263.00 |
+| Argentina          |  2780400.00 |
+| Kazakstan          |  2724900.00 |
++--------------------+-------------+
+10 rows in set (0.00 sec)
+
+5. What's the top 10 most populated cities?
+
+SELECT Name,Population FROM city 
+ORDER BY Population DESC LIMIT 10;
```
Comment (jasongwartz) [inline Week1/homework/world-queries.txt]: 👍

File: Week1/homework/world-queries.txt [hunk 5]
```diff
@@ -37,4 +39,199 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
...
+| Moscow            |    8389200 |
+| New York          |    8008278 |
++-------------------+------------+
+10 rows in set (0.00 sec)
+
+6. Select the top 3 countries by population that their names start with the letter ‘P’
+
+SELECT Name,Population FROM city 
+WHERE city.Name LIKE 'P%' 
+ORDER BY Population DESC LIMIT 3;
```
Comment (jasongwartz) [inline Week1/homework/world-queries.txt]: 👍

File: Week1/homework/world-queries.txt [hunk 6]
```diff
@@ -37,4 +39,199 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
...
+| Peking    |    7472000 |
+| Pusan     |    3804522 |
+| Pyongyang |    2484000 |
++-----------+------------+
+3 rows in set (0.00 sec)
+
+7. What is the population in Asia?
+
+SELECT Continent,SUM(Population) FROM country 
+WHERE country.Continent = 'Asia';
```
Comment (jasongwartz) [inline Week1/homework/world-queries.txt]: 👍

File: Week1/homework/world-queries.txt [hunk 7]
```diff
@@ -37,4 +39,199 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
...
++-----------+-----------------+
+| Asia      |      3705025700 |
++-----------+-----------------+
+1 row in set (0.00 sec)
+
+8. Which languages are spoken in the Region of ‘South America’
+
+SELECT country.Name,Language FROM countrylanguage 
+JOIN country ON country.Code = countrylanguage.Countrycode 
+WHERE country.Region = 'South America';
```
Comment (jasongwartz) [inline Week1/homework/world-queries.txt]: 👍

File: Week1/homework/world-queries.txt [hunk 8]
```diff
@@ -37,4 +39,199 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
...
+| Venezuela        | Warrau           |
++------------------+------------------+
+42 rows in set (0.00 sec)
+
+9. What are the languages spoken on all cities named ‘Barcelona’ (you may need to join 3 tables =) )
+
+SELECT city.Name,country.Name,Language FROM countrylanguage 
+JOIN city ON city.Countrycode = countrylanguage.Countrycode 
+JOIN country ON city.Countrycode = country.Code 
+WHERE city.Name = 'Barcelona';
```
Comment (jasongwartz) [inline Week1/homework/world-queries.txt]: 👍

Wow, when I was testing to check the correct answers, I did the query in the exact same way as you, except I didn't include `country.Name` in the results - I assumed all seven languages where from Barcelona, ESP. I had no idea some of them were from Venezuela!

File: Week1/homework/world-queries.txt (hunk 2 above)
Comment (jamsrahimi) [reply Week1/homework/world-queries.txt]: Hi @arasmr and @hackoor83, did you commit the fixes or comment on each other?

**Pull request closed without merging**